addon/adapters/user.js

```diff
@@ -8,11 +8,12 @@ export default OsfAdapter.extend({
 
         url = this.urlPrefix(url, this.buildURL(type, id, snapshot, 'findHasMany'));
 
-        // If fetching user nodes, will embed root and parent.
+        // If fetching user nodes, will embed root and parent. (@hmoco 12-27-17: why?)
         if (relationship.type === 'node') {
             url += '?embed=parent&embed=root';
-            if (snapshot.record.get('query-params')) {
-                url += '&' + Ember.$.param(snapshot.record.get('query-params'));
+            // TODO: revisit this, we shouldnt hard code any embeds
+            if (snapshot.record.get('__' + relationship.key + 'QueryParams')) {
+                url += '&' + Ember.$.param(snapshot.record.get('__' + relationship.key + 'QueryParams'));
             }
         } else if (snapshot.record.get('query-params')) {
```

Comment:
I'm pretty sure findHasMany is an overloaded adapter internal that was used by `user.query`. Not sure if `user.queryHasMany` circumvents that. Looking into it now.

Reply:
queryHasMany seems to not use the findHasMany. Is `query` deprecated? If so, I can just remove the method entirely.

Reply:
@aaxelb or @jamescdavis ^ ?

Reply:
Yeah, `user.query` is gone, replaced by `user.queryHasMany` (which bypasses ember-data almost entirely). I'm in favor of removing this.

Is there anywhere that actually uses the `query-params` thing? Because that should die in a fire.

Reply:
Once this is removed from the user adapter, nothing but our beloved `loadAll` :)

Reply:
(or at least that's what a quick global search returns)